src/Tribe/Commerce/Cart.php

```diff
@@ -317,19 +320,19 @@ public function update( $args ) {
 			}
 
 			try {
-				foreach ( $providers as $provider_tickets ) {
+				foreach ( $providers as $ticket_provider => $provider_tickets ) {
 					/**
 					 * Update tickets in cart for provider.
 					 *
-					 * The dynamic portion of the hook name, `$provider`, refers to the cart provider.
+					 * The dynamic portion of the hook name, `$ticket_provider`, refers to the cart provider.
```

Comment:
This now refers to the _ticket_ provider (unless for some reason it doesn't have one and gets the default cart provider).

Reply:
✅